### PR TITLE
Rspec feature login

### PR DESCRIPTION
**Pull request by haunc-0912**
==> Tittle: RSPEC Feature Login

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/16308

==> Attachment File:
Screenshot from 2019-08-22 09-48-40

### CODE REVIEW

**Comment by haunc-0912** [comment]
ready

File: spec/features/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  let!(:user) {FactoryBot.create :user, admin: false}
+
+  before do
+    visit root_path
+  end
+
+  context "Login success" do
+    scenario "role: user" do
+      fill_in "session[email]", with: user.email
+      fill_in "session[password]",	with: user.password
```
Comment (hungct-1121) [inline spec/features/login_spec.rb]:
```suggestion
      fill_in "session[password]", with: user.password
```

File: spec/features/login_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      click_button "commit"
+
+      expect(page).to have_text I18n.t "month_report.title.month_report"
+    end
+
+    scenario "role: admin" do
+      fill_in "session[email]", with: user_admin.email
+      fill_in "session[password]",	with: user_admin.password
```
Comment (hungct-1121) [inline spec/features/login_spec.rb]:
```suggestion
      fill_in "session[password]", with: user_admin.password
```

**Comment by haunc-0912** [comment]
ready

**Pull request merged**